src/components/template/Success-view.tsx

```diff
@@ -0,0 +1,271 @@
+'use client';
+
+import { useUser } from '@clerk/nextjs';
+import { Check, ChevronRight, Copy, FileText } from 'lucide-react';
+import Link from 'next/link';
+import { useEffect, useState } from 'react';
+
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/ui/tabs';
+import { getClientSecret } from '@/libs/actions/user';
+import type { SuccessViewProps } from '@/types/Template';
+
+import { IntegrationCodeViewer } from './handlebars-editor/IntegrationCodeViewer';
+
+export default function SuccessView({ templateId, templateName, templateSampleData, onViewDashboard, onCreateAnother }: SuccessViewProps) {
+  const [activeTab, setActiveTab] = useState('javascript');
+  const { user } = useUser();
+  const [secret, setSecret] = useState<string>('');
+  const [copied, setCopied] = useState<string | null>(null);
+
+  const handleSecret = async () => {
+    if (!user) {
+      return;
+    }
+    try {
+      const result = await getClientSecret(user.id);
+
+      if (!result) {
+        return;
+      }
+
+      setSecret(result.clientSecret);
+    } catch (error) {
+      console.error(`Failed to retrieve client Secret, ${error}`);
+    }
+  };
+
+  useEffect(() => {
+    handleSecret();
+  }, [user]);
+
+  // Parse the template sample data if available
+  const parsedSampleData = templateSampleData
+    ? (() => {
+        try {
+          return JSON.parse(templateSampleData as string);
+        } catch (e) {
+          console.error(e);
+          return {};
+        }
+      })()
+    : {};
+
+  const handleCopy = (code: string, language: string) => {
+    navigator.clipboard.writeText(code);
+    setCopied(language);
+    setTimeout(() => setCopied(null), 2000);
+  };
+
+  // Generate a formatted version of the sample data for code snippets
+  const formattedSampleData = Object.keys(parsedSampleData).length > 0
+    ? JSON.stringify(parsedSampleData, null, 2)
+    : `{}`;
+
+  // Code snippets for different languages
+  const codeSnippets = {
+    javascript: `// Using fetch API
+const response = await fetch('https://api.templify.com/convert', {
```

Comment:
use api.templify.com from a API_BASE_URL global constant
if not used this constant, use across the app and docs

Reply:
wrong url, our url is /templateid right?